studying.html

```diff
@@ -0,0 +1,80 @@
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/v4-shims.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Stencil+Display:wght@100&family=Roboto:wght@100&display=swap" rel="stylesheet">
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./styling/styles.css">
+        <link rel="stylesheet" href="./styling/studying.css">
+        <title>Code Journal</title>
+    </head>
+    <body>
+        <div class="content">
+            <nav class="navigation">
+                <ul>
+                    <li class="left-float"><img src="./images/nav-icon.jpeg" alt="Image of cute potato"></li>
+                    <li class="left-float nav-name"><a href="./index.html">Karolina</a></li>
+                    <li class="right-float"><a href="./about.html">About</a></li>
+                    <li class="right-float"><a href="./portfolio.html">Portfolio</a></li>
+                    <li class="right-float"><a href="./code-journal.html">Code Journal</a></li>
+                    <li class="right-float"><a href="./index.html">Home</a></li>
+                </ul>
+            </nav>
+            <header>
+                <h1>Studying Tips</h1>
+            </header>
+            <section>
+                <!-- #1 -->
+                <p style="font-size: 24px;">#1: Reading is not studying</p>
+                <p>Simply reading and re-reading texts or notes is not actively engaging in the material. It is simply re-reading your notes. Only ‘doing’ the readings for class is not studying. It is simply doing the reading for class. Re-reading leads to quick forgetting.</p>
+
+                <!-- #2 -->
+                <p style="font-size: 24px;">#2: Spacing out is good</p>
+                <p>One of the most impactful learning strategies is “distributed practice”—spacing out your studying over several short periods of time over several days and weeks (Newport, 2007). The most effective practice is to work a short time on each class every day. The total amount of time spent studying will be the same (or less) than one or two marathon library sessions, but you will learn the information more deeply and retain much more for the long term—which will help get you an A on the final. The important thing is how you use your study time, not how long you study. Long study sessions lead to a lack of concentration and thus a lack of learning and retention.
+                <br>
+                <p>Life Pro Tip: try the <a href="
+                    TomatoTimerhttps://tomato-timer.com">Pomodoro Technique</a></p>
```

Comment:
Careful of the url here. This causes a validation error.
```suggestion
                <p>Life Pro Tip: try the <a href="https://tomato-timer.com">Pomodoro Technique</a></p>
```